packages/cli/package.json

```diff
@@ -4,7 +4,10 @@
   "description": "On-Boarding Cli",
   "main": "index.js",
   "scripts": {
+    "build:post": "mkdir dist/constants/dashboard && cp src/constants/dashboard/index.html dist/constants/dashboard/index.html",
```

Comment:
@djabarovgeorge Decided to leave the "cp" part here, and added a new "build:prod" step instead of the current build. Which will probably mostly run in the CI environment. If needed later we can change to a more platform agnostic approach.

Reply:
Sure anyway I hope most of the users will use the docker-compose because it is more practical.